modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -24,6 +24,12 @@ def post_appointment(request_object_body)
           data: OpenStruct.new(response.body),
           meta: {}
         }
+      rescue Common::Exceptions::BackendServiceException => e
+        # TODO: Reevaluate the need to log clinic data three months after launch (6/15/20)
+        if e.key == 'VAOS_400'
```

Comment:
NITPICK: consider putting this logic in a method -- surprised it didn't trigger code climate, but it certainly will next time someone adds one or more lines to this.